test/runner/.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+	"root": true,
```

Comment:
Can we migrate ESLint to the flat config in Migrate as well?

Reply:
yes, eventually

Reply:
OK, I assume you mean a separate PR some time later.